openfl/federated/task/runner_tf.py

```diff
@@ -276,7 +288,8 @@ def get_tensor_dict(self, with_opt_vars=True):
         return {var.name: val for var, val in zip(
             variables, self.sess.run(variables))}
```

Comment:
Honestly, I do not understand how it is not `None` in any case.

Reply:
We define variables `self.opt_vars` and `self.fl_vars` at the heir class https://github.com/intel/openfl/blob/develop/openfl-workspace/tf_2dunet/src/tf_2dunet.py#:~:text=self.fl_vars%20%3D%20self.tvars%20%2B%20self.opt_vars